court_scraper/base/_last_date.py

```diff
@@ -0,0 +1,41 @@
+from datetime import timedelta, datetime
+from pandas.tseries.holiday import USFederalHolidayCalendar
+
+class LastDate:
```

Comment:
This function provides a library of tools for the case number discovery tool that will return dates to feed search pages. It does not conflict with the master.